internal/services/parse/resource.go

```diff
@@ -46,6 +46,13 @@ func NewResourceID(name, parentId, resourceType string) (ResourceId, error) {
 			azureResourceId = fmt.Sprintf("/subscriptions/%s", name)
 		case arm.TenantResourceType.String():
 			azureResourceId = "/"
+		case arm.ProviderResourceType.String():
```

Comment:
Persumably this wants to change? Simply merging into the `default` branch and check whether there is only one segment in the `azureResourceType` should suffice?